app/Http/Controllers/Auth/LoginController.php

```diff
@@ -39,6 +39,23 @@ public function __construct()
 		$this->middleware('guest', ['except' => 'logout']);
 	}
 
+	/**
+	 * Show the application's login form.
+	 *
+	 * @return \Illuminate\View\View
+	 */
+	public function showLoginForm()
+	{
+		$now = now();
+		$signupsOpen = App\Models\Product::select()
+			->where('slug', App\Models\Product::SLUG_WNL_ONLINE)
+			->whereDate('signups_start', '<', $now)
+			->whereDate('signups_end', '>', $now)
```

Comment:
```suggestion
			->whereDate('signups_end', '>=', $now)
```
I think the dates are inclusive, so users can sign up on the day of `signups_start` and the day of `signups_end`.